health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DrawerFieldComposer.js

```
@@ -289,70 +342,80 @@ const RenderField = ({ state, panelItem, drawerState, setDrawerState, updateLoca
+          <div
+            ref={switchRef}
+            className="drawer-container-tooltip"
+            onMouseEnter={() => setShowTooltip(true)}
+            onMouseLeave={() => setShowTooltip(false)}
+          >
+            {/* {disableFieldForMandatory(drawerState, panelItem, resourceData) && (
+            <span className="onhover-tooltip-text"> {t("MANDATORY_FIELD_PROPERTIES_DISABLE_HOVER_TEXT")}</span>
+          )} */}
+            {isDisabled && <TooltipPortal text={t("MANDATORY_FIELD_PROPERTIES_DISABLE_HOVER_TEXT")} targetRef={switchRef} visible={showTooltip} />}
+            <div style={{ display: "flex" }}>
               <label>{t(Digit.Utils.locale.getTransformedLocale(`FIELD_DRAWER_LABEL_${panelItem?.label}`))}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)